samples/protected-setting-sample/README.md

```diff
@@ -0,0 +1,43 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Protected settings sample
+
+This sample shows how ISV can create protected settings and input secret values in UIDefinition as well as consume them in ARM Template so that they can be passed to Kubernetes Extension Type as Configuration Protected Settings.
+
+Protected configuration settings help manage sensitive data securely. These settings typically include information such as passwords, API keys, certificates, or connection strings. Protected configuration settings are stored as secrets on the Kubernetes clusters. As a result, the key names of the protected configuration settings should be simple text and special characters should not be included. Protected configuration settings for an extension instance are stored for up to 48 hours in the cluster extension services. As a result, if the cluster remains disconnected during the 48 hours after the extension resource is created in Azure, the extension changes from a Pending state to a Failed state. To prevent this, it is recommended that ISVs bring clusters online regularly. 
```

Comment:
In the statement "To prevent this, it is recommended that ISVs bring clusters online regularly.", Change 'ISVs' to 'customers'